### PR TITLE
new_owner: switch on the new owner

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
switch on the new owner, it's the last pr of the owner refactor

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: tests/changefeed_error/run.sh [hunk 1]
```diff
@@ -129,21 +130,23 @@ function run() {
     fi
 
     ensure $MAX_RETRIES check_changefeed_mark_failed_regex http://${UP_PD_HOST_1}:${UP_PD_PORT_1} ${changefeedid} ".*CDC:ErrStartTsBeforeGC.*"
-    changefeed_info=$(ETCDCTL_API=3 etcdctl --endpoints=${UP_PD_HOST_1}:${UP_PD_PORT_1} get /tidb/cdc/changefeed/info/${changefeedid}|tail -n 1)
-    new_info=$(echo $changefeed_info|sed 's/"state":"failed"/"state":"normal"/g')
-    ETCDCTL_API=3 etcdctl --endpoints=${UP_PD_HOST_1}:${UP_PD_PORT_1} put /tidb/cdc/changefeed/info/${changefeedid} "$new_info"
+#    changefeed_info=$(ETCDCTL_API=3 etcdctl --endpoints=${UP_PD_HOST_1}:${UP_PD_PORT_1} get /tidb/cdc/changefeed/info/${changefeedid}|tail -n 1)
+#    new_info=$(echo $changefeed_info|sed 's/"state":"failed"/"state":"normal"/g')
+#    ETCDCTL_API=3 etcdctl --endpoints=${UP_PD_HOST_1}:${UP_PD_PORT_1} put /tidb/cdc/changefeed/info/${changefeedid} "$new_info"
+    cdc cli changefeed resume -c $changefeedid
```
Comment (leoppro) [inline tests/changefeed_error/run.sh:136]:
```suggestion
    cdc cli changefeed resume -c $changefeedid
```

File: tests/changefeed_finish/run.sh [hunk 2]
```diff
@@ -12,19 +12,21 @@ MAX_RETRIES=10
...
-    info=$(cdc cli changefeed query -c=$changefeed|sed "/has been deleted/d"|jq ".info")
-    if [[ ! "$info" -eq "null" ]]; then
-        echo "unexpected changefeed info $info, should be null"
-        exit 1
-    fi
+#    info=$(echo "$query"|sed "/has been deleted/d"|jq ".info")
+#    if [[ ! "$info" -eq "null" ]]; then
+#        echo "unexpected changefeed info $info, should be null"
+#        exit 1
+#    fi
```
Comment (leoppro) [inline tests/changefeed_finish/run.sh:27]:
```suggestion
```

File: tests/kill_owner_with_ddl/run.sh [hunk 3]
```diff
@@ -25,7 +25,11 @@ function kill_cdc_and_restart() {
     work_dir=$2
     cdc_binary=$3
     MAX_RETRIES=10
-    cdc_pid=$(curl -s http://127.0.0.1:8300/status|jq '.pid')
+    status=$(curl -s http://127.0.0.1:8300/status)
+    cdc_pid=$(echo "$status"|jq '.pid')
+    echo "$status"
+    echo "$cdc_pid"
```
Comment (leoppro) [inline tests/kill_owner_with_ddl/run.sh:31]:
```suggestion
```

File: cdc/processor/processor.go [hunk 4]
```diff
@@ -655,22 +644,13 @@ func (p *processor) addTable(ctx cdcContext.Context, tableID model.TableID, repl
...
-	globalCheckpointTs := p.changefeed.Status.CheckpointTs
-
-	if replicaInfo.StartTs < globalCheckpointTs {
-		log.Warn("addTable: startTs < checkpoint",
-			cdcContext.ZapFieldChangefeed(ctx),
-			zap.Int64("tableID", tableID),
-			zap.Uint64("checkpoint", globalCheckpointTs),
-			zap.Uint64("startTs", replicaInfo.StartTs))
-	}
 	table, err := p.createTablePipeline(ctx, tableID, replicaInfo)
```
Comment (leoppro) [inline cdc/processor/processor.go:654]:
```suggestion

 	globalCheckpointTs := p.changefeed.Status.CheckpointTs

 	if replicaInfo.StartTs < globalCheckpointTs {
 		log.Warn("addTable: startTs < checkpoint",
 			cdcContext.ZapFieldChangefeed(ctx),
 			zap.Int64("tableID", tableID),
 			zap.Uint64("checkpoint", globalCheckpointTs),
 			zap.Uint64("startTs", replicaInfo.StartTs))
 	}
	table, err := p.createTablePipeline(ctx, tableID, replicaInfo)
```

File: cdc/processor/manager.go [hunk 5]
```diff
@@ -100,6 +100,11 @@ func (m *Manager) Tick(stdCtx context.Context, state orchestrator.ReactorState)
 			if changefeedState.Status.AdminJobType.IsStopState() || changefeedState.TaskStatuses[captureID].AdminJobType.IsStopState() {
 				continue
 			}
+			// the processor should start after one table added to this processor at least
```
Comment (liuzix) [inline cdc/processor/manager.go:103]:
```suggestion
			// the processor should start after at least one table has been added to this capture
```

File: cdc/processor/processor.go [hunk 6]
```diff
@@ -193,6 +193,14 @@ func (p *processor) tick(ctx cdcContext.Context, state *model.ChangefeedReactorS
 	return p.changefeed, nil
 }
 
+func (p *processor) checkChangefeedNormal() bool {
```
Comment (liuzix) [inline cdc/processor/processor.go:196]: Can you add comment for this function?

File: cdc/processor/processor.go [hunk 7]
```diff
@@ -538,8 +525,9 @@ func (p *processor) checkTablesNum(ctx cdcContext.Context) error {
 		if opt != nil && opt[tableID] != nil {
 			continue
 		}
-		if p.initialized {
-			log.Warn("the table should be listen but not, already listen the table again, please report a bug", zap.Int64("tableID", tableID), zap.Any("replicaInfo", replicaInfo))
+		log.Info("start to listen the table immediately", zap.Int64("tableID", tableID), zap.Any("replicaInfo", replicaInfo))
```
Comment (liuzix) [inline cdc/processor/processor.go:528]:
```suggestion
		log.Info("start to listen to the table immediately", zap.Int64("tableID", tableID), zap.Any("replicaInfo", replicaInfo))
```

File: tests/ddl_reentrant/run.sh [hunk 8]
```diff
@@ -103,7 +103,7 @@ function ddl_test() {
 
     echo $ddl > ${WORK_DIR}/ddl_temp.sql
     ensure 10 check_ddl_executed "${WORK_DIR}/cdc.log" "${WORK_DIR}/ddl_temp.sql" true
-    ddl_start_ts=$(grep "Execute DDL succeeded" ${WORK_DIR}/cdc.log|tail -n 1|grep -oE '"start_ts\\":[0-9]{18}'|awk -F: '{print $(NF)}')
+    ddl_start_ts=$(grep "Execute DDL succeeded" ${WORK_DIR}/cdc.log|tail -n 1|grep -oE '"StartTs\\":[0-9]{18}'|awk -F: '{print $(NF)}')
```
Comment (amyangfei) [inline tests/ddl_reentrant/run.sh:106]: Why change the keyword from `start_ts` to `StartTs`

Comment (leoppro) [reply tests/ddl_reentrant/run.sh:106]: because the log changed

Comment (leoppro) [reply tests/ddl_reentrant/run.sh:106]: https://github.com/pingcap/ticdc/blob/master/cdc/changefeed.go#L741 in the old owner
https://github.com/pingcap/ticdc/blob/master/cdc/owner/async_sink.go#L137 in the new owner

**Review by liuzix — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**